index.html

```diff
@@ -52,8 +52,16 @@
 
 		<!-- ## SLIDE TEXT ## -->
 		<div id="qcSlideText">
-			<h1 id="slidecaption">Write/Speak/Code 2017 conference</h1>
-			<p id="slidedesc">Join us for 4 intense days of workshops, seminars, and talks.<br>Let's build the blueprint that will take your career to the next level!</p>
+			<h1 id="slidecaption"><span class="wsc">Write/Speak/Code</span> 2017 conference</h1>
+			<div id="slidedesc">
```

Comment:
@jarmstrng I guess it's up to you, I'd just as rather not create a new class (or target part of an h1 tag) if the only difference is that a piece of the title vs the entire title is a bit smaller.